src/createAnimatedComponent.js

```diff
@@ -344,6 +355,12 @@ export default function createAnimatedComponent(Component) {
       return props;
     }
 
+    _obtainViewName() {
+      const viewConf =
+        this._component?.viewConfig || this._component?.root?.viewConfig;
```

Comment:
this uses some internal structures of react-native and deserves at least a comment on what we're accessing. What is `root`? What is `viewConfig`? What may break here if that internal structure changes?

Reply:
fixed dafa7117cea37a58342516e6aad0e03bbf6eecc8